packages/babel-parser/src/parser/location.js

```diff
@@ -98,6 +98,11 @@ export const Errors = Object.freeze({
   MissingUnicodeEscape: "Expecting Unicode escape sequence \\uXXXX",
   MixingCoalesceWithLogical:
     "Nullish coalescing operator(??) requires parens when mixing with logical operators",
+  ModuleAttributeInvalidValue:
+    "Only string literals are allowed as module attributes values",
```

Comment:
```suggestion
    "Only string literals are allowed as module attribute values",
```

Reply:
Thank you @xtuc . That seems correct to me and was a mistake at my end.